.circleci/config.yml

```diff
@@ -109,6 +109,11 @@ commands:
         - run:
             name: Create reports directory
             command: mkdir reports && mkdir reports/test-results
+  add-jest-snapshot-dir:
+      steps:
+        - run:
+            name: Create snapshots directory
+            command: mkdir __device-tests__/__image_snapshots__/__diff_output__
```

Comment:
That is a good question, I'm not sure why `__device-tests__` has that name, for the image snapshots folders it is how the library generates them but we could customize the folder names according to the `jest-image-snapshot` [docs](https://github.com/americanexpress/jest-image-snapshot#%EF%B8%8F-api). Maybe we could rename them like:

- `__image_snapshots__` -> `image-snapshots`
- `__diff_output__` -> `diff`

What do you think?

Reply:
Ah, OK. Thanks for sharing that context via the docs. Maybe the double underscore pattern is merely a way to have generated directories grouped atop file lists with default name sort. 🤷🏻‍♂️ 

I do not have a strong opinion. The names you propose are clear and consistent. Those look good to me. 

If we'd prefer to avoid additional configuration, I think it is fine to retain the default names.

Reply:
Since we are already updating some settings in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5456/files#diff-ffabdf74f7609e8b6a7e64b0ddceff3ea0457ac950ada6ba1c7e39cd3bfe08c3R11 we can include these name customizations as well. I'll update the PR 😃

Reply:
Updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5456/commits/86a2046f3c0ef3b756916bd6509dc90faaad297d